### DISCUSSION TITLE
New features request: Blitter and (Emulated) Numeric Coprocessor

### DISCUSSION THREAD

**Discussion by basic-xl** [picocomputer/community#94] [Q&A]
I'm relatively sure 6502 accessible blitter functionality could be handled by the ria, if it isn't already. At least I'm hoping. But I'm also intrigued about the possibility of an emulated numeric coprocessor support, for super fast FP Math routines. That'd be a great tool for someone interested in creating a new BASIC, for example 😉 Thanks everyone!

**Comment by grakoczy** [comment]
I'm not sure if blitter could be implemented but it would be great if we could have fill rectangle and line drawing implemented on the pico side.
This would make drawing way faster. 
I know we have great graphics library by @vruumllc but it relies on CPU for those operations. 
I wonder how much faster it could work, but I guess it will be quite faster.
